ProcessCIW/ProcessCIW.csproj

```diff
@@ -39,7 +39,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="DocumentFormat.OpenXml">
-      <HintPath>..\..\..\Downloads\DocumentFormat.OpenXml.dll</HintPath>
+      <HintPath>..\..\..\..\Downloads\DocumentFormat.OpenXml.dll</HintPath>
```

Comment:
There was a broken reference to a dll located outside the project directory that I fixed. Prob should move the dll into the project directory to avoid pathing issues.